resources-fre/strings/strings.xml

```diff
@@ -89,16 +89,16 @@
 	<string id="Sat">Sam</string>
 
 	<string id="Jan">Jan</string>
-	<string id="Feb">Fev</string>
+	<string id="Feb">Fév</string>
 	<string id="Mar">Mar</string>
 	<string id="Apr">Avr</string>
 	<string id="May">Mai</string>
 	<string id="Jun">Juin</string>
 	<string id="Jul">Juil</string>
-	<string id="Aug">Aout</string>
+	<string id="Aug">Août</string>
```

Comment:
Many thanks for all these translations.

The accented `û` is not included in one or more date fonts, so appears incorrectly. Could you fix this?

![image](https://user-images.githubusercontent.com/6435959/103682380-ba9a1380-4f80-11eb-913d-9b357cc2884e.png)

Reply:
Do you know which font is used or do simply cut and paste a standard 'u' and add a caret over it in PaintShopPro to make a û?

Reply:
I think I've catched them all. Also fixed the ':' for low power mode Venu Sq.